<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="OB-Yogesh" failures="2" tests="2" name="tests.VATOnlyInvoice_DefaultProfile" time="29.554" errors="0" timestamp="2021-01-25T14:31:00 IST" skipped="0">
  <testcase classname="tests.VATOnlyInvoice_DefaultProfile" name="GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive1" time="11.506">
    <failure message="Unable to locate object: By.xpath: //LI[@id=&#039;ctl00_SideMenu1_incomeMenu&#039;]/A" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_SideMenu1_incomeMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.VATOnlyInvoice.Click_SalesTAB(VATOnlyInvoice.java:93)
at tests.VATOnlyInvoice_DefaultProfile.GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive1(VATOnlyInvoice_DefaultProfile.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive1 -->
  <system-out/>
  <testcase classname="tests.VATOnlyInvoice_DefaultProfile" name="GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive2" time="18.048">
    <failure message="Unable to locate object: By.xpath: //LI[@id=&#039;ctl00_SideMenu1_incomeMenu&#039;]/A" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_SideMenu1_incomeMenu']/A
at org.testng.Assert.fail(Assert.java:99)
at pages.VATOnlyInvoice.Click_SalesTAB(VATOnlyInvoice.java:93)
at tests.VATOnlyInvoice_DefaultProfile.GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive2(VATOnlyInvoice_DefaultProfile.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlClickSalesTABClickNewTABClickVATOnlyInvoicePositiveSelectCustomerNamePositive2 -->
  <system-out/>
</testsuite> <!-- tests.VATOnlyInvoice_DefaultProfile -->
